### PR TITLE
Figure fixes

### LINKED ISSUE #32: Reviewer #1 - Figure/Table Todos
Reviewer 1 suggests the following changes to tables and figures:
- [x] Table 1: What is the organizing principle behind the order of the databases? Would it be better to order them alphabetically?
- [x] Table 3: Maybe define or spell out the Types of Sentences.
- [x] Figure 5b: Font too small

### LINKED ISSUE #34: Reviewer #2 - Figure 1 Feedback
Reviewer 2 comments that the knowledge graph example doesn't have arrows for some of the relationships:

>**Figure 1 doesn't show the relationship direction, For example, "causes", "binds" and other relations don't clearly specify the source and destination nodes which can be confusing.  Ideally, a knowledge graph should show that. 

Suggested fix is to include an example of another knowledge graph that has arrows.

### PR DESCRIPTION
**Pull request by danich1**
Closes #32 and closes #34 
Main changes here are bigger font for figure 5b, sorted and updated table entries and change example knowledge graph to include arrows.
Coming next are text changes.

### CODE REVIEW

File: content/images/figures/hetionet_metagraph_subgraph.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...outline" cx="0" cy="0" r="32.5" fill="#008F00" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Anatomy</text></g><g class="node" transform="translate(458.343297060915,140.77998893983684)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#A5ABB6" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Symptom</text></g><g class="node" transform="translate(-233.3344148499738,35.601194363297935)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#FFFB00" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#000000"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#000000" font-family="sans-serif"> SideEffect</text></g></g></svg>
```
Comment (cgreene) [inline content/images/figures/hetionet_metagraph_subgraph.svg:1]: Are some of these bidirectional or undirected relationships? In particular, I'm wondering about things like associates and binds.

File: content/02.introduction.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ We also compare assorted approaches for applying knowledge graphs to solve biome
 Lastly, we conclude on the practicality of knowledge graphs and point out future applications that have yet to be explored.
 
 ![
-A metagraph (schema) of the heterogeneous network used in the Rephetio project [@doi:10.7554/eLife.26726].
-This undirected network depicts pharmacological and biomedical information.
+A subset of the knowledge graph used in the Rephetio project [@doi:10.7554/eLife.26726].
```
Comment (cgreene) [inline content/02.introduction.md:30]: If it's a subset of het.io then it probably needs to use the same directionality (if any) that het.io uses. I think some edges are directional but not all.

**Review by cgreene — COMMENTED** [review]
I had some questions re: directionality. Maybe those changes will be clearer to me when the text changes land, so we could wait for those before coming back to this.

**Comment by danich1** [comment]
For ease of use I moved these changes to their respective sections. Closing this PR and here are references to those PR:
- Figure 1 #42 
- Table reordering #44 
- Table acronyms spelled out #45

**Pull request closed without merging**